2/src/styles/_media-mobile.scss

```diff
@@ -0,0 +1,227 @@
+@media screen and (max-width: 767px) {
```

Comment:
the task was use `mobile-first`  approach

Reply:
overwrite